### DISCUSSION TITLE
add <kbd> tag style

### DISCUSSION THREAD

**Discussion by ToBinio** [rust-lang/crates.io#6420] [Feature Requests]
I recently published a small cargo-plugin where i wanted to explain in the README how it is used and what the keys do.

Sadly the current crates.io Markdown renderer does not highlight the \<kbd> html-tag in any real way.

in my opion it should look something like the github style -> <kbd>kbd</kbd>

**Comment by Turbo87** [comment]
sounds like a reasonable suggestion. would you be interested in trying to implement it yourself? unfortunately, we are currently quite swamped with other work so I can't say when we would otherwise have the time to implement this 😅

**Reply by ToBinio** [reply]
shure i would like to try.

is there anything special i should think about / pay attention?

**Reply by Turbo87** [reply]
there are two things to do:

1) ensure that our markdown renderer (see https://github.com/rust-lang/crates.io/blob/master/cargo-registry-markdown/lib.rs) is allowing `<kbd>` tags by e.g. adding a unit test for it if it does not exist yet.

2) add the necessary styles to https://github.com/rust-lang/crates.io/blob/master/app/components/rendered-html.module.css to make sure that `<kbd>` look pretty :)

**Reply by ToBinio** [reply]
just wanted so say thanks for you helpfull and quick answers 😊
